middlewares/replace_path.go

```diff
@@ -18,3 +18,8 @@ func (s *ReplacePath) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r.URL.Path = s.Path
 	s.Handler.ServeHTTP(w, r)
 }
+
+// SetHandler sets handler
+func (s *ReplacePath) SetHandler(Handler http.Handler) {
```

Comment:
I've updated the code to remove it from this particular place, so hopefully it's good to go. ;)